install/infra/terraform/gke/main.tf

```diff
@@ -152,6 +152,29 @@ resource "google_container_node_pool" "workspaces" {
   }
 }
 
+resource "google_sql_database_instance" "gitpod" {
+  name = "sql-${var.name}"
+  database_version = "MYSQL_5_7"
+  region = "${var.region}"
+  settings {
+      tier = "db-n1-standard-2"
```

Comment:
This is a good point, ideally we can make this configurable and use a cheaper one for tests. I was basically just translating the [reference architecture](https://www.gitpod.io/docs/self-hosted/latest/reference-architecture/production-ready-single-cluster) here. I was thinking that since our test setup is ephemeral the extra cost we pay here is quite small. May be I am wrong, and it adds up. I will make a note for this for a follow-up